lib/nerves_hub_cli/api.ex

```diff
@@ -157,11 +157,12 @@ defmodule NervesHubCLI.API do
       is_atom(ca_store) and !is_nil(ca_store) ->
         ca_store.ca_certs()
 
-      Code.ensure_loaded?(CAStore) ->
-        CAStore.file_path()
-        |> File.read!()
-        |> X509.from_pem()
-        |> Enum.map(&X509.Certificate.to_der/1)
+      # CAStore uses the priv directory, which does not work with escripts
```

Comment:
this is an interesting issue.

Reply:
It looks like the OS-supplied certs are returned in the `true` clause. It seems like this code could be completely deleted.

Reply:
you are right, i guess we can just state that you need to keep your os supplied certs up to date